ale_linters/cloudformation/cfn_python_lint.vim

```diff
@@ -0,0 +1,33 @@
+" Author: Yasuhiro Kiyota <yasuhiroki.duck@gmail.com>
+" Description: Support cfn-python-lint for AWS Cloudformation template file
+
+function! ale_linters#cloudformation#cfn_python_lint#Handle(buffer, lines) abort
+    " Matches patterns line the following:
+    "
+    " sample.template.yaml:96:7:96:15: [E3012] Property Resources/Sample/Properties/FromPort should be of type Integer
```

Comment:
Is that an end line number and column in the output? You could include those in the Dictionary below too as `end_col` and `end_lnum`.

Reply:
Thank you.
I didn't realize I can use `end_xxx`. I'll fix it.